tests/protractor/e2e/medic-reporter-modal.js

```diff
@@ -0,0 +1,104 @@
+var utils = require('../utils');
+
+describe('Medic Reporter modal', function() {
+  'use strict';
+
+  var forms = JSON.parse('{ "OFF": { "meta": { "code": "OFF", "label": { "en": "Disable Notifications" } }, "fields": { "patient_id": { "labels": { "tiny": { "en": "ID" }, "description": { "en": "Patient ID" }, "short": { "en": "ID" } }, "position": 0, "flags": { "input_digits_only": true }, "length": [ 5, 5 ], "type": "string" }, "notes": { "labels": { "tiny": { "en": "r", "sw": "r" }, "description": { "en": "Reason" }, "short": { "en": "Reason" } }, "position": 1, "length": [ 2, 100 ], "type": "string" } } }, "ON": { "meta": { "code": "ON", "label": { "en": "Enable Notifications" } }, "fields": { "patient_id": { "labels": { "tiny": { "en": "ID" }, "description": { "en": "Patient ID" }, "short": { "en": "ID" } }, "position": 0, "flags": { "input_digits_only": true }, "length": [ 5, 5 ], "type": "string" } } } }');
```

Comment:
Any reason this isn't actual JSON? I'd at least recommend breaking the string up into lots of them (and putting them back together with `.join('')` to get rid of the giant horizontal scrolling.